pytest_tests/tests/object/test_object_api.py

```diff
@@ -281,6 +292,133 @@ def _generate_filters_expressions(attrib_dict: dict[str, str]):
                 )
                 assert sorted(expected_oids) == sorted(result)
 
+    @pytest.mark.parametrize("operator", ["GT", "GE", "LT", "LE"])
+    @pytest.mark.parametrize(
+        "object_size",
+        [pytest.lazy_fixture("simple_object_size"), pytest.lazy_fixture("complex_object_size")],
+        ids=["simple object", "complex object"],
+    )
+    def test_object_search_with_numeric_queries(
+        self, default_wallet: NodeWallet, container: str, object_size: int, operator: str
+    ):
+        objects = []
+        for numeric_value in OBJECT_NUMERIC_VALUES:
+            file_path = generate_file(object_size)
+
+            objects.append(
+                {
+                    "numeric_value": numeric_value,
```

Comment:
`"numeric_value"` key string used multiple times, can we define a const for it?

Reply:
Thank you, there was already a const for it, i just didn't use it here. Fixed.